doc/semantics-of-the-publishing-api.md

```diff
@@ -0,0 +1,406 @@
+
+The `description` captures a short explanation of the content item. It is sent
+downstream to the content store and may be displayed at the discretion of the
+front-end application, when the content item is rendered.
+
+---
+
+### need_ids
+
+Example: *["1234", "1235"]*
+
+Required: No
+
+The `needs_ids` are the identifiers of [user needs](https://www.gov.uk/design-principles)
+that are entered through the [Maslow Application](https://github.com/alphagov/maslow).
+They are passed through to the content store, untouched by the pipeline. The
+front-end applications can then use the `needs_ids` to present pages to users,
+such as [this one](https://www.gov.uk/info/overseas-passports).
+
+---
+
+### locale
+
+Example: *en*
+
+Required: No
+
+The `locale` is the language in which the content item is written. Front-end
+applications will optionally provide this string when querying the content store
+in order to retrieve content items in a given `locale`.
+
+A list of valid locales can be viewed in the Publishing API [here](https://github.com/alphagov/publishing-api/blob/master/config/application.rb#L32-L37).
+This field is not required and if a `locale` is not provided, it will be set to
+*en* automatically.
+
+---
+
+### phase
+
+Examples: *alpha, beta, live*
+
+Required: No
+
+The `phase` is an optional field that may be used to indicate the ‘Service
+Design Phase’ of the content item. The phase must be one of either *alpha*,
+*beta* or *live*. If the `phase` is not specified, the Publishing API will
+default it to *live*.
+
+Content items will be published to the content store regardless of their
+`phase`.  If a content item has a `phase` of either *alpha* or *beta*, a visual
+component will be added to the front-end to show this (assuming the front-end
+application for that format supports this).
+
+There is more information on what each of the phases mean [here](https://www.gov.uk/service-manual/phases).
+
+---
+
+### access_limited
+
+Example: *{ users: ["bf3e4b4f-f02d-4658-95a7-df7c74cd0f50"] }*
+
+Required: No
+
+The `access_limited` field determines who should be granted access to the
+content item before it has been published. At the point of publish the content
+item is public and everyone has access to it. This field should be used when a
+content item needs to be drafted, but it should not be visible on
+content-preview (and become public knowledge) until a publish occurs.
+
+The value of this field should be set to a JSON object that contains the key
+*users* and a value that is an array of [UUIDs](https://en.wikipedia.org/wiki/Universally_unique_identifier).
+These are the users that should be granted access to the content item. At
+present, the only supported key is *users*.  If `access_limited` is not set, no
+access restriction will be placed on the content item.
+
+When front-end applications make requests to the content store, they must supply
+the user they are making the request on behalf of if the content item is
```

Comment:
Given that is implemented in the content store rather than in the Publishing API, I'd be tempted to either link to the content store OR say that "items with access_limited are limited to only the user IDs appearing in the list" (since the details of the content-store implementation may change).